MobSF/utils.py

```diff
@@ -445,11 +445,11 @@ def get_adb():
             adb = 'adb'
             if platform.system() == 'Darwin':
                 adb_dir = os.path.join(settings.TOOLS_DIR, 'adb/mac/')
-                subprocess.call(['chmod', '777', adb_dir])
+                os.chmod(adb_dir, 0o744)
```

Comment:
777 or 774 is enough?

Reply:
in my test yes but it means that the user who do the install is the same who launch MobSF